ha/util/conf_store.py

```diff
@@ -39,6 +41,8 @@ def __init__(self, conf_store_req: bool = True):
         """Init method"""
         if conf_store_req:
             self._confstore = ConfigManager.get_confstore()
+        # Load cluster config
+        ConfigManager._safe_load(CLUSTER_GLOBAL_INDEX, CLUSTER_CONF)
```

Comment:
Once. keys will be stored in consul which HA mini provisioning, this step will not be needed. Mariyappan confirmed.